spec.html

```diff
@@ -42161,16 +42161,17 @@ <h1>Static Semantics: Early Errors</h1>
             It is a Syntax Error if any source text matches this rule.
           </li>
         </ul>
+        <p>Additionally, the rules for the following productions are modified with the addition of the <ins>highlighted</ins> text:</p>
```

Comment:
For the Block and Switch errors below, this PR changes the affected sections to include errors which are not modified by this PR, presumably to make it clear that those rules are untouched. NonemptyClassRanges and NonemptyClassRangesNoDash also have multiple errors, but you haven't copied them here.